jax_triton/pallas/pallas_call.py

```diff
@@ -56,14 +59,20 @@
 
 pallas_call_p.def_impl(partial(xla.apply_primitive, pallas_call_p))
 
-def _maybe_dynamic_slice(start_idx, block_shape, value):
+def _maybe_dynamic_slice(start_idx, block_shape, value, is_indexing):
```

Comment:
It seems like is_indexing can be None? Wouldn't that interact weirdly with the indexing below?

Reply:
Or does that happen only when start_idx is None? Perhaps add an assertion?

Reply:
Yea it should only happen when `start_idx` is None. I'll add the assertion.